agent/utils.py

```diff
@@ -56,17 +59,30 @@ def prepare_target(message: m.Message) -> tuple[str, int]:
         raise NotImplementedError
 
 
-def initialize_msf_rpc() -> msfrpc.MsfRpcClient:
-    """Start msfrpcd and connect to it
+def start_msfrpcd() -> None:
+    """Start msfrpcd
```

Comment:
if it this method doesnt have args or a return type just remove them from the docstring